plugins/inlinetoolbar/plugin.js

```diff
@@ -110,6 +122,61 @@
 					that.attach( element, false );
 				} ) );
 			};
+
+			/**
+			 * Hides the inline toolbar, detaches listeners and moves the focus back to the editable.
+			 */
+			CKEDITOR.ui.inlineToolbarView.prototype.detach = function() {
+				this._detachListeners();
+				this.hide();
+			};
+
+			/**
+			 * Adds an item from the specified definition to the inline toolbar.
+			 *
+			 * @method
+			 * @param {String} name The menu item name.
+			 * @param {Object} definition The menu item definition.
+			 */
+			CKEDITOR.ui.inlineToolbarView.prototype.addMenuItem = function( name, definition ) {
```

Comment:
I have asked this to be fixed in https://github.com/ckeditor/ckeditor-dev/pull/992#discussion_r142139001